bats/conflict-detection.bats

```diff
@@ -241,12 +241,12 @@ SQL
     dolt checkout add-table
     dolt sql <<SQL
 CREATE TABLE test2 (
-  pk BIGINT NOT NULL COMMENT 'tag:0',
-  c1 BIGINT COMMENT 'tag:1',
-  c2 BIGINT COMMENT 'tag:2',
-  c3 BIGINT COMMENT 'tag:3',
-  c4 BIGINT COMMENT 'tag:4',
-  c5 BIGINT COMMENT 'tag:5',
+  pk BIGINT NOT NULL COMMENT 'tag:10',
```

Comment:
Bats tests needed to be updated to either
1) Eliminate a tag conflict created by a tag annotation
2) Add a tag annotation to prevent random tag generation

Reply:
Unless it's a test specifically about tag generation, bats tests we shouldn't specify tag numbers when creating tables. Existing examples were mostly copy-pasted (since the output include tag number comments).